platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,480 @@
+
+###### Characteristics:
+
+* Application teams create new capabilities as needed to meet pressing needs
+* A central team provides a register of available shared services across the organization
+* Loose standards, such as requiring an automatable API and usage docs, are applied to capabilities
+* Infrastructure as Code is used to allow easier traceability of deployed services
+* Audits for compliance regulations such as PCI DSS or HIPPA are enabled through the service inventories
+* Migration and upgrade work is tracked against a burndown chart enabling the organization to track rate of compliance and time until completion
+* Tracking does not indicate level of support; often upgrades at this stage are still manual and bespoke
+
+###### Examples Scenarios:
+
+* PostgreSQL 11 is going EOL by the end of the year. The organization is aware of which databases require upgrade and are scheduling the work on each team’s backlog to complete.
+
+##### Level 3, Scalable - Centrally enabled 
+
+Platforms and capabilities are not only centrally registered but also centrally orchestrated. Platform teams take responsibility for understanding the broad needs of the organization and prioritize work across platform and infrastructure teams accordingly. Those responsible for a capability are expected to not only maintain it technically, but also provide standard user experiences for integrating the capability with other related services around the organization, ensure secure and reliable use, and even provide observability.
+
+Standard processes for creating and evolving new capabilities exist enabling anyone in the organization to contribute a solution that meets expectations. Continuous delivery processes for platform capabilities and features enable regular rollout and rollback. Large changes are planned and coordinated as they would be for customer-facing product changes.
+
+###### Characteristics:
+
+* Application teams request services from platform teams first before creating them
+* New services must adhere to standard practices such as standard interfaces, documentation, and governance
+* Upgrade processes are documented and consistent across versions and services
+* Where the capability provider does not manage an upgrade, they provide tooling and support to the users for minimal impact
+
+###### Examples Scenarios:
+
+* The organization is going to upgrade to RHEL 9. In doing so, each application team needs to validate that their software continues to work. In order to enable this testing phase the centralized compute team is generating test environments for each team with the correct software and OS versions.
```

Comment:
Nit-pik: setting up seems a better word here since it's commonly used with environments. 
```suggestion
* The organization is going to upgrade to RHEL 9. In doing so, each application team needs to validate that their software continues to work. In order to enable this testing phase the centralized compute team is setting up test environments for each team with the correct software and OS versions.
```